Simulation.java

```diff
@@ -46,7 +50,8 @@
     private final static long RANDOM_SEED = 1;
     private final static int DEFAULT_NUM_ITERATIONS = 10;
     private final static int DEFAULT_SCALE_FACTOR = 5;
-    private final static Logger LOG = LoggerFactory.getLogger(Simulation.class);
+    private final static String DEFAULT_CONFIG_YAML = "config/config.yaml";
+    private final static Logger LOGGER = LoggerFactory.getLogger(Simulation.class);
```

Comment:
Why did we change the name to `LOGGER`? I think `LOG` has been the standard elsewhere in grakn.

Reply:
Because this clashes with `LOG` the enum for a logarithmic sampling function. Perhaps we should change that instead though